src/iOS/Xamarin.iOS/ViewControllers/SamplesViewController.cs

```diff
@@ -93,13 +94,14 @@ public override async void RowSelected(UITableView tableView, NSIndexPath indexP
                         _controller.ParentViewController.View.Add(_loadPopup);
 
                         // Ensure data present
-                        await DataManager.EnsureSampleDataPresent(sample);
+                        await DataManager.EnsureSampleDataPresent(_sample);
 
                         // Hide progress overlay
                         _loadPopup.Hide();
                     }
 
-                    var control = (UIViewController)SampleManager.Current.SampleToControl(sample);
+                    var control = (UIViewController)SampleManager.Current.SampleToControl(_sample);
+                    control.NavigationItem.RightBarButtonItem = new UIBarButtonItem("\uD83D\uDEC8", UIBarButtonItemStyle.Plain, ViewSampleReadme);
```

Comment:
I recommend taking the info button from the native iOS samples: https://github.com/Esri/arcgis-runtime-samples-ios/tree/master/arcgis-ios-sdk-samples/Images.xcassets/InfoIcon.imageset